src/app/space/create/deployments/apps/deployment-status-icon.component.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  Component,
+  Input,
+  OnDestroy,
+  OnInit
+} from '@angular/core';
+
+import {
+  Observable
+} from 'rxjs';
+
+import { CpuStat } from '../models/cpu-stat';
+
+const ICON_OK = 'pficon-ok';
+const ICON_WARN = 'pficon-warning-triangle-o';
+const ICON_ERR = 'pficon-error-circle-o';
+const MSG_OK = 'Everything is ok.';
+const STAT_THRESHOLD = .6;
+@Component({
+  selector: 'deployment-status-icon',
+  templateUrl: 'deployment-status-icon.component.html',
+  styleUrls: ['./deployment-status-icon.component.less']
+})
+export class DeploymentStatusIconComponent implements OnDestroy, OnInit {
+  @Input() cpuDataStream: Observable<CpuStat>;
+
+  iconClass: String;
+  toolTip: String;
+  subscriptions: Array<Subscription> = [];
+
+  constructor() {
+    this.iconClass = ICON_OK;
+    this.toolTip = MSG_OK;
+  }
+
+  ngOnDestroy(): void {
+    this.subscriptions.forEach(sub => sub.unsubscribe());
+  }
+
+  ngOnInit(): void {
+    this.subscriptions.push(this.cpuDataStream.subscribe((stat) => {
+      this.changeStatus(stat);
+    }));
+  }
+
+  changeStatus(stat: CpuStat) {
+    this.iconClass = ICON_OK;
+    this.toolTip = MSG_OK;
+    if (stat.used / stat.total > STAT_THRESHOLD) {
```

Comment:
The Stat definition has `'used` and `quota`. I assume you mean `quota` in your references to `total`.

Reply:
Thanks. This has been addressed and pushed.

Reply:
@chrislessard 

The builder has two more errors now:

ERROR in [at-loader] ./src/app/space/create/deployments/apps/deployment-card.component.spec.ts:67:38 

    TS2304: Cannot find name 'CpuStat'.



ERROR in [at-loader] ./src/app/space/create/deployments/apps/deployment-status-icon.component.ts:29:24 

    TS2304: Cannot find name 'Subscription'.